src/Hocon.Configuration/Config.cs

```diff
@@ -123,12 +138,6 @@ public virtual Config GetConfig(string path)
         public virtual Config GetConfig(HoconPath path)
         {
             var value = GetNode(path);
-            if (Fallback != null)
```

Comment:
Same as above, since `GetNode` is loading node from `Root`, which already has all fallback info.